plugins/lookup/onepassword.py

```diff
@@ -508,17 +509,22 @@ def _parse_field(self, data_json, field_name, section_title=None):
 
                 # If the field name doesn't exist in the section, match on the value of "label"
                 # then "id" and return "value"
-                if field.get("label") == field_name:
+                if field.get("label").lower() == field_name:
```

Comment:
If `field.get()` returns `None`, then this will result in an exception.

How about creating a helper function `def lower(value):` which returns `None`, and applies `lower` elsewise?

Reply:
I considered this, but `label` and `id` are part of the schema and are expected to be in the field data.

I'll just default to an empty string to avoid the potential exception.

I actually thought about a helper function for the `field_name` and `section_title` parameters. Since `field_name` defaults to a string, I just tested `section_title`. From a unit-testing perspective, I helper function would probably be better for those parameters.